packages/orchestrator-ui-components/src/components/WfoBadges/WfoSubscriptionSyncStatusBadge/WfoSubscriptionSyncStatusBadge.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react';
+
+import { useOrchestratorTheme } from '../../../hooks';
+import { WfoBadge } from '../WfoBadge';
+
+export type WfoSubscriptionSyncStatusBadgeProps = {
+    insync: boolean;
+};
+
+export const WfoSubscriptionSyncStatusBadge: FC<
+    WfoSubscriptionSyncStatusBadgeProps
+> = ({ insync }) => {
+    const { theme, toSecondaryColor } = useOrchestratorTheme();
+
+    const getBadgeColorFromStatus = (insync: boolean) => {
```

Comment:
This returning more than the color so I would call it getBadgeProperties or something